dev-docs/modules/a1MediaRtdProvider.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: page_v2
+title: A1Media RTD Module
+display_name: A1Media RTD Module
+description: A1Media Real Time Data Module
+page_type: module
+module_type: rtd
+module_code : a1MediaRtdProvider
+enable_download : true
+vendor_specific: true
+sidebarType : 1
+---
+
+# A1Media RTD Module
+
+{: .alert.alert-warning :}
+Disclosure: This module loads external code that is not open source and has not been reviewed by Prebid.org.
+
+The A1Media RTD module loads A1Media script for obtains user segments, and provides user segment data to bid-requests.<br />
+to get user segments, you will need a1media script customized for site.
```

Comment:
Suggested wording:

"The A1Media RTD module loads a script for obtaining A1Media user segments, providing the user segment data to bid-requests."

Reply:
Thanks for the suggestion!
we changed as your suggestion

Reply:
Cool, but you left the overlapping sentence "<br />to get user segments, you will need a1media script customized for site." Please remove that part?

Reply:
We removed overwrapped part.
Please tell me if there is something more to improve